relayer/src/channel.rs

```diff
@@ -704,7 +704,7 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Channel<ChainA, ChainB> {
     pub fn build_chan_open_init_and_send(&self) -> Result<IbcEvent, ChannelError> {
         let dst_msgs = self.build_chan_open_init()?;
 
-        let tm = TrackedMsgs::new(dst_msgs, "create channel");
+        let tm = TrackedMsgs::new(dst_msgs, "create channel with ChannelOpenInit");
```

Comment:
I would be in favor of keeping this message short, since it's used as prefix on many log messages. Suggestion below:

```suggestion
        let tm = TrackedMsgs::new(dst_msgs, "ChannelOpenInit");
```